components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -85,6 +93,17 @@ public TemplateManagerService() throws TemplateManagerServiceException, RuleTemp
                 loadAndSaveAnalyticsSolutions(solutionType);
             }
         }
+        if (configReader.isSiddhiAppManagerEnabled()) {
+            isSiddhiAppDeployerEnabled = true;
+            siddhiAppManagerDeploymentPattern = configReader.getSiddhiAppDeploymentPattern();
+            if (siddhiAppManagerDeploymentPattern.equals(DEPLOYMENT_PATTERN_LOAD_BALANCING)) {
+                siddhiAppDeployer = new LoadBalancingDeployer();
+                siddhiAppDeployer.init(configReader, siddhiAppApiHelper);
```

Comment:
Arent we enabling `load balancing pattern` by default if deployment pattern is not mentioned?

Reply:
Fixed this.